src/components/Button/src/Button.vue

```diff
@@ -272,8 +411,13 @@ export default {
 		min-width: max-content;
 	}
 
+	&.size_expandable {
+		min-height: 48px;
+		padding: 5px 20px;
+	}
+
 	&.size_small {
-		height: 32px;
+		height: 36px;
```

Comment:
this is a weird choice, like i mentioned in another comment pretty much all of our margins, paddings, and explicit widths & heights are multiples of 8, and prior to this change the button heights were 32px (8*4) for small, 48px (8*6) for medium, and 64px (8*8) for large. Changing the small size from 32px to 36px breaks this convention so i think it warrants some justification, and i think we should prefer keeping the small size 32px if possible.